rasa/core/policies/rule_policy.py

```diff
@@ -34,6 +34,11 @@
     USER_INTENT_SESSION_START: ACTION_SESSION_START_NAME,
 }
 
+RULES = "rules"
+NEGATIVE_RULES = "negative_rules"
+NO_VALIDATION = "no_validation"
+NO_ACTIVE_FORM = "no_active_form"
```

Comment:
The name is somewhat confusing, how about
```suggestion
DO_NOT_PREDICT_FORM_ACTION = "no_active_form"
```